react/components/ProductPrice/Installments.js

```diff
@@ -7,11 +8,44 @@ import PricePropTypes from './propTypes'
 
 /** Installments component */
 export default class Installments extends Component {
+
+  static defaultProps = {
+    classes: {
+      root: null,
+      installmentValue: null,
+      interestRate: null
+    }
+
+  }
+
+  static propTypes = {
+    /** Product installments to be displayed */
+    installments: PricePropTypes.installments,
+    /** Pages editor config to display labels */
+    showLabels: PropTypes.bool.isRequired,
+    /** react-intl function to format the prices*/
+    formatNumber: PropTypes.func.isRequired,
+    /** Options to be passe to the formatNumber function*/
+    currencyOptions: PropTypes.shape({
+      style: PropTypes.string.isRequired,
+      currency: PropTypes.string.isRequired,
+      minimumFractionDigits: PropTypes.number.isRequired,
+      maximumFractionDigits: PropTypes.number.isRequired,
+    }).isRequired,
+    /** Classes to be applied in the Installments Component */
+    classes: PropTypes.shape({
+      root: PropTypes.string,
```

Comment:
move these to single prop e.g. `className`, `installmentClass`, `interestRateClass`